Cargo.toml

```diff
@@ -10,3 +10,5 @@ members = [
 [patch.crates-io]
 prost-derive = { git = "https://github.com/danburkert/prost" }
 
+[patch."https://github.com/tower-rs/tower-h2"]
+tower-h2 = { git = "https://github.com/hawkw/tower-h2", branch = "eliza/new-tokio" }
```

Comment:
This branch just updates tower-h2's examples, there aren't significant functional changes. However, the examples have to compile with the new tokio, or else the dependency fails to compile.

I'll open a PR to merge this branch.

Reply:
tower-rs/tower-h2#27